From 8590206686cf4c953d60d9b6d8e2f1b5dbd39233 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 7 Feb 2018 18:47:09 +0800
Subject: [PATCH] mfd/fusb302: fix compile warning

drivers/mfd/fusb302.c:1164:3: warning: memset used with length
equal to number of elements without multiplication by element
size [-Wmemset-elt-size]

Change-Id: I33ac6408bc4cee656c7f18948d418527619b3d6b
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/mfd/fusb302.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mfd/fusb302.c b/drivers/mfd/fusb302.c
index f95b8851445d..4f5b1abd282b 100644
--- a/drivers/mfd/fusb302.c
+++ b/drivers/mfd/fusb302.c
@@ -1161,7 +1161,7 @@ static int vdm_send_discoverysvid(struct fusb30x_chip *chip, int evt)
 	switch (chip->vdm_send_state) {
 	case 0:
 		set_vdm_mesg(chip, VDM_DISCOVERY_SVIDS, VDM_TYPE_INIT, 0);
-		memset(chip->vdm_svid, 0, 12);
+		memset(chip->vdm_svid, 0, sizeof(chip->vdm_svid));
 		chip->vdm_svid_num = 0;
 		chip->tx_state = 0;
 		chip->vdm_send_state++;
-- 
2.35.3

